src/lib/MuncherSettings.js

```diff
@@ -581,7 +581,7 @@ const MuncherSettings = {
       {
         name: "add-effects",
         isChecked: game.settings.get(SETTINGS.MODULE_ID, "munching-policy-add-effects"),
-        description: "Dynamically generate effects (equipment only). (Requires DAE)",
+        description: "Dynamically generate ActiveEffects (equipment only) for any equipment that modify properties of actors? (Requires DAE and is not recommended unless you are aiming for a highly automated game.)",
```

Comment:
This is not the case here, most of these effects that are applied via this check box are not highly automate.d

Reply:
Then DAE is not needed.

Reply:
I would say that most folks probably do want this enabled, this is often items that give bonuses to saves etc. Not what I would call highly automated. There is very little high automation for items.

Reply:
Is the DDB importer unable to create effects on items without DAE? Why is that? Unless the configuration of the effects requires DAE, I see no reason for it being required.